[@mantine/charts] RadarChart: Add Tooltips and Dots

LilyStilson opened the pull request:
In the latest Mantine version `7.17.5` RadarChart does not have any tooltips. This PR implements them with a Storybook demo of custom tooltip.

Tooltip example:
<img width="476" alt="image" src="https://github.com/user-attachments/assets/040a695b-001e-41d2-99c0-2d1632ed440c" />

Custom tooltip demo:
<img width="434" alt="image" src="https://github.com/user-attachments/assets/ab9790f9-64f5-466f-9eef-b4a21125b6d2" />

LilyStilson commented:
Added dots to match `LineChart`
<img width="424" alt="image" src="https://github.com/user-attachments/assets/04f3d120-ae53-47f5-858b-41290648cd07" />

LilyStilson commented:
Btw, I couldn't figure out how to put cursor under dots. Looks like its default behavior in Recharts...

rtivital commented:
Please fix the following:
1. By default, the tooltip should be disabled and dots should not be visible. It is needed to avoid unexpected breaking changes. Please also add a separate demo to showcase tooltip feature.
2. When dot is active, it is being overlaid by the axis, see screenshot. It must render above the axis the same way as an inactive dot.

<img width="711" alt="Снимок экрана 2025-04-28 в 09 13 52" src="https://github.com/user-attachments/assets/6130bd37-94da-49c4-a1ac-05821affb0eb" />

LilyStilson commented:
@rtivital Should dots be enabled automatically when `withTooltip` is specified?

LilyStilson commented:
Fixed cursor line being above dots, apparently the only way to do this - put tooltip above `{radars}`.

<img width="909" alt="image" src="https://github.com/user-attachments/assets/cab2fea7-2d55-43ab-b324-768f6a8f3e65" />

Also, added demo for default tooltip.

rtivital commented:
Thanks!
